crates/cxx-qt-gen/src/generator/cpp/property/mod.rs

```diff
@@ -71,17 +72,37 @@ pub fn generate_cpp_properties(
 mod tests {
```

Comment:
appears this needs to be at least `pub(crate)` otherwise other places cannot reach it